testsrc/main/mondrian/xmla/XmlaBasicTest.java

```diff
@@ -744,12 +744,12 @@ public void testExecuteAliasWithSharedDimension()
             + "<Schema name=\"foodmart-xmla-alias-bug\">\n"
             + "  <Dimension name=\"Customers\">\n"
             + "    <Hierarchy hasAll=\"true\" allMemberName=\"All Customers\" primaryKey=\"customer_id\">\n"
-            + "      <Table schema=\"foodmart\" name=\"customer\"/>\n"
+            + "      <Table name=\"customer\"/>\n"
```

Comment:
@mkambol 
When schema attribute is used, mondrian will generate sql which runs query against 'foodmart.customer' table which is ok for mysql, but not for Oracle. Otherwise, if this attribute is not used, it will infer schema from connection properties and create correct query for both databases. I've checked foodmart and steelwheels shemas: they don't contain this attribute in table element. I guess it was a copy paste error from some bad example.